[DDW-296] Daedalus review fixes

DeeJayElly opened the pull request:
This PR fixes all UI specific issues found during the last review of Daedalus. 
Lists of fixed issues: 
1. https://docs.google.com/document/d/1iFkI644fXMWgvFIGVFwLrvLzrvjnUyGO0FlmA2pPPPQ/edit
2. https://docs.google.com/document/d/1Wj6_TNos1jglVfKK2EV23zZ9PopLeKRuzpucE58PsNE/edit


---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1597241906053700)

---

## Review Checklist

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

DeeJayElly commented:
@a-rukin @mioriohk Pls recheck document all issues https://docs.google.com/document/d/1iFkI644fXMWgvFIGVFwLrvLzrvjnUyGO0FlmA2pPPPQ/edit

mioriohk commented:
![deletewallet](https://user-images.githubusercontent.com/67595081/90377950-aa84a480-e070-11ea-862c-826ad6d3d1a6.gif)
Hi @DeeJayElly , I am not sure if this is intentional or not. During 'Delete wallet', when I selected the checkbox, I was unable to deselect it. If I am not mistaken, the way how checkbox works is that the user is able to select or unselect it.

nikolaglumac commented:
> ![deletewallet](https://user-images.githubusercontent.com/67595081/90377950-aa84a480-e070-11ea-862c-826ad6d3d1a6.gif)
> Hi @DeeJayElly , I am not sure if this is intentional or not. During 'Delete wallet', when I selected the checkbox, I was unable to deselect it. If I am not mistaken, the way how checkbox works is that the user is able to select or unselect it.

@mioriohk that is by design!

ManusMcCole commented:
Hi @DeeJayElly. For Wallet creation the spinner after hitting confirm is still a light colour for Shelley Testnet where in the document it specifys that it should be the same colour as the text
![image](https://user-images.githubusercontent.com/56027085/90499101-06b6f980-e141-11ea-9787-86e06e8087ac.png)

mioriohk commented:
![Annotation 2020-08-18 114838_3](https://user-images.githubusercontent.com/67595081/90504575-32d67880-e149-11ea-8357-349db75ec8b9.gif)
![Annotation 2020-08-18 114742_2](https://user-images.githubusercontent.com/67595081/90504581-3538d280-e149-11ea-9bd4-42cd9be84339.gif)
![Annotation 2020-08-18 114604_1](https://user-images.githubusercontent.com/67595081/90504583-35d16900-e149-11ea-86fb-6b01775f0fc5.gif)
@DeeJayElly Text for progress bar for themes yellow, white and ITN don't match specification. Tested on build 14193

mioriohk commented:
![syncrewalletdelegate](https://user-images.githubusercontent.com/67595081/90508915-69fc5800-e150-11ea-954f-d602be8a6590.gif)
@DeeJayElly I restored a wallet and when it's still syncing, I went to delegate screen, check dropdown and could see the syncing animation. In the spec, an indication of it syncing should be shown but it's not. Tested on build 14193

DeeJayElly commented:
> ![Annotation 2020-08-18 114838_3](https://user-images.githubusercontent.com/67595081/90504575-32d67880-e149-11ea-8357-349db75ec8b9.gif)
> ![Annotation 2020-08-18 114742_2](https://user-images.githubusercontent.com/67595081/90504581-3538d280-e149-11ea-9bd4-42cd9be84339.gif)
> ![Annotation 2020-08-18 114604_1](https://user-images.githubusercontent.com/67595081/90504583-35d16900-e149-11ea-86fb-6b01775f0fc5.gif)
> @DeeJayElly Text for progress bar for themes yellow, white and ITN don't match specification. Tested on build 14193

@mioriohk thats old build, can you pls retest on Build #14195

mioriohk commented:
@DeeJayElly For build 14195, only yellow is correct. white and ITN are still wrong. For white, both progress bar and text colour are wrong. For ITN, it's the text colour
> > @DeeJayElly Text for progress bar for themes yellow, white and ITN don't match specification. Tested on build 14193
> 
> @mioriohk thats old build, can you pls retest on Build #14195

DeeJayElly commented:
> > ![Annotation 2020-08-18 114838_3](https://user-images.githubusercontent.com/67595081/90504575-32d67880-e149-11ea-8357-349db75ec8b9.gif)
> > ![Annotation 2020-08-18 114742_2](https://user-images.githubusercontent.com/67595081/90504581-3538d280-e149-11ea-9bd4-42cd9be84339.gif)
> > ![Annotation 2020-08-18 114604_1](https://user-images.githubusercontent.com/67595081/90504583-35d16900-e149-11ea-86fb-6b01775f0fc5.gif)
> > @DeeJayElly Text for progress bar for themes yellow, white and ITN don't match specification. Tested on build 14193
> 
> @mioriohk thats old build, can you pls retest on Build #14195

fixed

mioriohk commented:
@DeeJayElly Looks good now, thank you
> > > @DeeJayElly Text for progress bar for themes yellow, white and ITN don't match specification. Tested on build 14193
> > 
> > 
> > @mioriohk thats old build, can you pls retest on Build #14195
> 
> fixed

mioriohk commented:
> ![syncrewalletdelegate](https://user-images.githubusercontent.com/67595081/90508915-69fc5800-e150-11ea-954f-d602be8a6590.gif)
> @DeeJayElly I restored a wallet and when it's still syncing, I went to delegate screen, clicked dropdown and couldn't see the syncing animation. In the spec, an indication of it syncing should be shown but it's not. Tested on build 14193

Retested. Still not fixed on build 14201

gnpf commented:
@DeeJayElly Here's a summary of some issues I found (some of them are not listed on the original document, so apologies if this is not on the scope of this PR):

`Note: Screenshots are linked`

1 - The text on the screen "Rewards redemption not available" (Piggy bank with no wallet) is not on the correct color for themes [**Yellow**](https://drive.google.com/file/d/1xsuYGD3TXGMWIXsSiZTQucCe8HveG_nN/view?usp=sharing) and **[White](https://drive.google.com/file/d/1jgwwHpx6E3z_o0HFqkZ1rtQGprNxGEWh/view?usp=sharing)**.

2 - On _Delegation Center - Rewards_ the spinner for syncing wallets is not visible on **[Cardano](https://drive.google.com/file/d/1YNY1vbx7aYMG2FRdhrmC_NPCth2zvRjd/view?usp=sharing)** and **[Light Blue](https://drive.google.com/file/d/14weL-K6mQxVaN9BM_uwOi8HNyApG2cSB/view?usp=sharing)** themes. (**[Here's an example](https://drive.google.com/file/d/1_REh30xm_lKU53VcIplTVb86tC2XUdPU/view?usp=sharing)** of how it looks on the rest of the themes)

3 - There are a couple of issues ([**screenshot 1**](https://drive.google.com/file/d/167fp7b9NycCUmUyMwxQYYqKmAedNvBHb/view?usp=sharing), [**screenshot 2**](https://drive.google.com/file/d/1CeSFmVM0Q9vQ7USoiFTKlB9TFuUhChgo/view?usp=sharing)) that were reported on [DDW-262](https://jira.iohk.io/browse/DDW-262). Not sure if they need to be fixed here too or if this is being addressed on the corresponding [PR](https://github.com/input-output-hk/daedalus/pull/2128). Anyway, I wanted to mention them just in case this is a merging issue.

ManusMcCole commented:
Hi @DeeJayElly. I still see the spinner as a lighter colour on the newest builds 
<img width="700" alt="Screenshot 2020-08-19 at 10 25 34" src="https://user-images.githubusercontent.com/56027085/90618772-557b9680-e208-11ea-9a37-60132dfb2cc1.png">

ManusMcCole commented:
Hi @DeeJayElly . I still see the spinner as a light colour and not the colour of the buttton text in the newest builds
<img width="670" alt="Screenshot 2020-08-19 at 10 38 07" src="https://user-images.githubusercontent.com/56027085/90618849-6b895700-e208-11ea-8e9a-c0881ef1b000.png">

gnpf commented:
@DeeJayElly Here are a few issues I found:

`Note: Screenshots are linked and were taken on a different version, but were reproduced on 14206`

1 - The [**color of the text**](https://drive.google.com/file/d/1U1gNw3GiSWfOxLMsm3WsfSPmhDlJqwM2/view?usp=sharing) is incorrect for the dropdown on "Rewards redemption" (White theme).

2 - The [**color of the tag**](https://drive.google.com/file/d/1H6FMlBxYZ40EXcEOB18aCDb6bHyT0Jxu/view?usp=sharing) on "Rewards redemption" is incorrect (Light Blue theme). [**Here**](https://app.zeplin.io/project/57acfaa7436a750316161a8b/screen/5ea9865471f61ebb869e4d94) is the Zeplin reference, @a-rukin please validate if this is correct.

DeeJayElly commented:
> @DeeJayElly Here are a few issues I found:
> 
> `Note: Screenshots are linked. Some were taken on a different version but were reproduced on 14206`
> 
> 1 - The [**color of the text**](https://drive.google.com/file/d/1U1gNw3GiSWfOxLMsm3WsfSPmhDlJqwM2/view?usp=sharing) is incorrect for the dropdown on "Rewards redemption" (White theme). **[Here](https://app.zeplin.io/project/59b94291478ed0294c8db870/screen/5f352747a990ae7f05f9127e)** is the Zeplin reference.
> 
> 2 - The [**color of the tag**](https://drive.google.com/file/d/1H6FMlBxYZ40EXcEOB18aCDb6bHyT0Jxu/view?usp=sharing) on "Rewards redemption" is incorrect (Light Blue theme). [**Here**](https://app.zeplin.io/project/57acfaa7436a750316161a8b/screen/5ea9865471f61ebb869e4d94) is the Zeplin reference, @a-rukin please validate if this is correct.
> 
> 3 - **[This issue](https://github.com/input-output-hk/daedalus/pull/2152#issuecomment-675409175)** with the progression bar that @mioriohk reported before is still happening for **[Dark Cardano](https://drive.google.com/file/d/1Miz0Jg6XInFvhoeuZR-BlKRPv4lcVSI9/view?usp=sharing)** theme. **[Here](https://app.zeplin.io/project/59b94291478ed0294c8db870/screen/5a0583be5abbf66a3a6b1c77)** is the Zeplin reference.
> 
> 4 - The background color for the dropdown on "Rewards redemption" is incorrect for the [**Dark Cardano**](https://drive.google.com/file/d/1T1e-UDGmSF6L_9qMJgm0DedJMCAGHs4L/view?usp=sharing) theme. **[Here](https://app.zeplin.io/project/59b94291478ed0294c8db870/screen/5f352747a990ae7f05f9127e)** is the Zeplin reference.

fixed

DeeJayElly commented:
> Hi @DeeJayElly . I still see the spinner as a light colour and not the colour of the buttton text in the newest builds
> <img alt="Screenshot 2020-08-19 at 10 38 07" width="670" src="https://user-images.githubusercontent.com/56027085/90618849-6b895700-e208-11ea-8e9a-c0881ef1b000.png">

fixed

DeeJayElly commented:
> @DeeJayElly Here's a summary of some issues I found (some of them are not listed on the original document, so apologies if this is not on the scope of this PR):
> 
> `Note: Screenshots are linked`
> 
> 1 - The text on the screen "Rewards redemption not available" (Piggy bank with no wallet) is not on the correct color for themes [**Yellow**](https://drive.google.com/file/d/1xsuYGD3TXGMWIXsSiZTQucCe8HveG_nN/view?usp=sharing) and **[White](https://drive.google.com/file/d/1jgwwHpx6E3z_o0HFqkZ1rtQGprNxGEWh/view?usp=sharing)**.
> 
> 2 - On _Delegation Center - Rewards_ the spinner for syncing wallets is not visible on **[Cardano](https://drive.google.com/file/d/1YNY1vbx7aYMG2FRdhrmC_NPCth2zvRjd/view?usp=sharing)** and **[Light Blue](https://drive.google.com/file/d/14weL-K6mQxVaN9BM_uwOi8HNyApG2cSB/view?usp=sharing)** themes. (**[Here's an example](https://drive.google.com/file/d/1_REh30xm_lKU53VcIplTVb86tC2XUdPU/view?usp=sharing)** of how it looks on the rest of the themes)
> 
> 3 - There are a couple of issues ([**screenshot 1**](https://drive.google.com/file/d/167fp7b9NycCUmUyMwxQYYqKmAedNvBHb/view?usp=sharing), [**screenshot 2**](https://drive.google.com/file/d/1CeSFmVM0Q9vQ7USoiFTKlB9TFuUhChgo/view?usp=sharing)) that were reported on [DDW-262](https://jira.iohk.io/browse/DDW-262). Not sure if they need to be fixed here too or if this is being addressed on the corresponding [PR](https://github.com/input-output-hk/daedalus/pull/2128). Anyway, I wanted to mention them just in case this is a merging issue.

All of the stated are fixed besides part of the codebase which handles tooltips - number 3.

That part will be resolved with Tooltip task https://jira.iohk.io/browse/DDW-262

DeeJayElly commented:
> @DeeJayElly Here are a few issues I found:
> 
> `Note: Screenshots are linked. Some were taken on a different version but were reproduced on 14206`
> 
> 1 - The [**color of the text**](https://drive.google.com/file/d/1U1gNw3GiSWfOxLMsm3WsfSPmhDlJqwM2/view?usp=sharing) is incorrect for the dropdown on "Rewards redemption" (White theme). **[Here](https://app.zeplin.io/project/59b94291478ed0294c8db870/screen/5f352747a990ae7f05f9127e)** is the Zeplin reference.
> 
> 2 - The [**color of the tag**](https://drive.google.com/file/d/1H6FMlBxYZ40EXcEOB18aCDb6bHyT0Jxu/view?usp=sharing) on "Rewards redemption" is incorrect (Light Blue theme). [**Here**](https://app.zeplin.io/project/57acfaa7436a750316161a8b/screen/5ea9865471f61ebb869e4d94) is the Zeplin reference, @a-rukin please validate if this is correct.
> 
> 3 - **[This issue](https://github.com/input-output-hk/daedalus/pull/2152#issuecomment-675409175)** with the progression bar that @mioriohk reported before is still happening for **[Dark Cardano](https://drive.google.com/file/d/1Miz0Jg6XInFvhoeuZR-BlKRPv4lcVSI9/view?usp=sharing)** theme. **[Here](https://app.zeplin.io/project/59b94291478ed0294c8db870/screen/5a0583be5abbf66a3a6b1c77)** is the Zeplin reference.
> 
> 4 - The background color for the dropdown on "Rewards redemption" is incorrect for the [**Dark Cardano**](https://drive.google.com/file/d/1T1e-UDGmSF6L_9qMJgm0DedJMCAGHs4L/view?usp=sharing) theme. **[Here](https://app.zeplin.io/project/59b94291478ed0294c8db870/screen/5f352747a990ae7f05f9127e)** is the Zeplin reference.

Fixed @gnpf

DeeJayElly commented:
> Hi @DeeJayElly . I still see the spinner as a light colour and not the colour of the buttton text in the newest builds
> <img alt="Screenshot 2020-08-19 at 10 38 07" width="670" src="https://user-images.githubusercontent.com/56027085/90618849-6b895700-e208-11ea-8e9a-c0881ef1b000.png">

Recheck it now @ManusMcCole

ManusMcCole commented:
Hi @DeeJayElly. Im still seeing the loading icon after creating a wallet on Shellley tesnet as a lighter colour than the buttons text
<img width="772" alt="Screenshot 2020-08-20 at 11 33 54" src="https://user-images.githubusercontent.com/56027085/90762808-8bd71580-e2dd-11ea-8cca-b0d971dde72c.png">

ManusMcCole commented:
Hi @DeeJayElly .The wallet spinner to indicate that a wallet is currently loading in Delegation Centre is only showing up for Dark Blue theme. Also doesnt not show up in Shelley Testnet

gnpf commented:
@DeeJayElly Most of the issues are fixed. Here a list of some of them that still reproduce and some new findings:

`Note: Screenshots are linked.`

1 - The **X** button for the "Rewards redemption not available" screen has a different design from other screens. Reproducible on [**all themes**.](https://drive.google.com/file/d/1ZTz2EFJp3KCahbcKOneVqUoNBXFYV_Bg/view?usp=sharing)

2 - Issue # 2 from **[this](https://github.com/input-output-hk/daedalus/pull/2152#issuecomment-676367327)** comment is still reproducible. **[Here](https://drive.google.com/file/d/1KTKEfxz5yxmEsMmJFbnDCMiejpiqweUe/view?usp=sharing)** is a new screenshot.

3 - Not sure about the color of the text for **[this](https://drive.google.com/file/d/1MV6_kdE6vzH4-xrqxuMy79TCK4TU2i9k/view?usp=sharing)** screen on the White theme, I couldn't find the expected design. @a-rukin can you advise on this, please?

a-rukin commented:
@DeeJayElly did you fix last doc comments? (not marked as done there + not fixed in last commit)

DeeJayElly commented:
> @DeeJayElly did you fix last doc comments? (not marked as done there + not fixed in last commit)

rechecking

mioriohk commented:
Hi @DeeJayElly , During wallet deletion (attached gif), I found out that the spinner colour and the text colour don't match for yellow and white themes but are correct for the others. Tested on 14224
![deletewallet1](https://user-images.githubusercontent.com/67595081/90788422-b33dda80-e2fd-11ea-92e0-7a6b7643fc99.gif)

gnpf commented:
@DeeJayElly I can still reproduce this issues:

`Note: Screenshots are linked and they were taken from a different version.`

1 - The [**color of the tag**](https://drive.google.com/file/d/1H6FMlBxYZ40EXcEOB18aCDb6bHyT0Jxu/view?usp=sharing) on "Rewards redemption" is incorrect (Light Blue theme). [**Here**](https://app.zeplin.io/project/57acfaa7436a750316161a8b/screen/5ea9865471f61ebb869e4d94) is the Zeplin reference

2 - Not sure about the color of the text for **[this](https://drive.google.com/file/d/1MV6_kdE6vzH4-xrqxuMy79TCK4TU2i9k/view?usp=sharing)** screen on the White theme.

@a-rukin Can you advise on these?

a-rukin commented:
> @DeeJayElly I can still reproduce these issues:
> 
> `Note: Screenshots are linked and they were taken from a different version.`
> 
> 1 - The [**color of the tag**](https://drive.google.com/file/d/1H6FMlBxYZ40EXcEOB18aCDb6bHyT0Jxu/view?usp=sharing) on "Rewards redemption" is incorrect (Light Blue theme). [**Here**](https://app.zeplin.io/project/57acfaa7436a750316161a8b/screen/5ea9865471f61ebb869e4d94) is the Zeplin reference
> 
> 2 - Not sure about the color of the text for **[this](https://drive.google.com/file/d/1MV6_kdE6vzH4-xrqxuMy79TCK4TU2i9k/view?usp=sharing)** screen on the White theme.
> 
> @a-rukin Can you advise on these?

Good catches @gnpf 
can you add this to the doc? 
https://docs.google.com/document/d/1Wj6_TNos1jglVfKK2EV23zZ9PopLeKRuzpucE58PsNE/edit
I'm not sure if @DeeJayElly will fix anything else in this PR

mioriohk commented:
@DeeJayElly @a-rukin Hi guys, if we find anything else. How would you like us to deal with it? Are you going to create a new card if one hasn't been created? Please advise. Thank you 👍

a-rukin commented:
> @DeeJayElly @a-rukin Hi guys, if we find anything else. How would you like us to deal with it? Are you going to create a new card if one hasn't been created? Please advise. Thank you 👍

my suggestion is to put stuff here https://docs.google.com/document/d/1Wj6_TNos1jglVfKK2EV23zZ9PopLeKRuzpucE58PsNE/edit 
so this August version would be the most relevant one

a-rukin reviewed (approved):
Great job! 👍

nikolaglumac commented:
@ManusMcCole @gnpf please review this one - we need to merge it today if everything is OK now.

nikolaglumac commented:
@daniloprates please review this one 🙏

ManusMcCole commented:
Hi @DeeJayElly. As per Sashas comment "Delete” button text should be dark #121326 for all themes". I see it as a light colour for themes Light Blue, White and Yellow 
<img width="825" alt="Screenshot 2020-09-02 at 12 17 14" src="https://user-images.githubusercontent.com/56027085/91975101-c8abff00-ed16-11ea-8889-5a3f43231fb0.png">
<img width="702" alt="Screenshot 2020-09-02 at 12 19 44" src="https://user-images.githubusercontent.com/56027085/91975114-cc3f8600-ed16-11ea-8d67-c1e16a21636b.png">

a-rukin commented:
> Hi @DeeJayElly. As per Sashas comment "Delete” button text should be dark #121326 for all themes". I see it as a light colour for themes Light Blue, White and Yellow
> <img alt="Screenshot 2020-09-02 at 12 17 14" width="825" src="https://user-images.githubusercontent.com/56027085/91975101-c8abff00-ed16-11ea-8889-5a3f43231fb0.png">
> <img alt="Screenshot 2020-09-02 at 12 19 44" width="702" src="https://user-images.githubusercontent.com/56027085/91975114-cc3f8600-ed16-11ea-8d67-c1e16a21636b.png">

wait wait wait... not for all themes)))
only for **dark-cardano/itn/flight/testnet** themes, which are one theme with different accent color

ManusMcCole reviewed (approved):
Good Job. Lots of great improvements in this PR 👍

gnpf commented:
@DeeJayElly All the fixes look good, I just found a few small issues. Let me know if you can include this on this PR or if I should create a new card for them:

- There is a vertical scrollbar on some dialogs (not reproducible on last production build): [create wallet dialog,
](https://drive.google.com/file/d/1Bz-HL5b34fBXoKHT1ybaO52k46vqVmNG/view?usp=sharing) [restore wallet dialog.
](https://drive.google.com/file/d/1QNzkoSXF5SwmtaSEbvTSfKrfSeziyHo6/view?usp=sharing)

- The spinner is still a different color from the text on the [delegation dialog](https://drive.google.com/file/d/1TgYYFtOBC74FibI5zaYs7-MIKXT65tX_/view?usp=sharing) (Shelley Testnet)

nikolaglumac commented:
There is an issue with recovery phrase validation messages on the "Wallet restoration" wizard which I will fix in the scope of this PR...

DeeJayElly commented:
> @DeeJayElly All the fixes look good, I just found a few small issues. Let me know if you can include this on this PR or if I should create a new card for them:
> 
> * There is a vertical scrollbar on some dialogs (not reproducible on last production build): [create wallet dialog,
>   ](https://drive.google.com/file/d/1Bz-HL5b34fBXoKHT1ybaO52k46vqVmNG/view?usp=sharing) [restore wallet dialog.
>   ](https://drive.google.com/file/d/1QNzkoSXF5SwmtaSEbvTSfKrfSeziyHo6/view?usp=sharing)
> * The spinner is still a different color from the text on the [delegation dialog](https://drive.google.com/file/d/1TgYYFtOBC74FibI5zaYs7-MIKXT65tX_/view?usp=sharing) (Shelley Testnet)

Fixed @gnpf

daniloprates commented:
About this one:

![image](https://user-images.githubusercontent.com/1504716/92136387-59034600-ede2-11ea-89dc-346da7ff8a4d.png)

This is how it's looking on Storybook. Waiting for node sync to check in the actual app, but it's worth having a look:

![image](https://user-images.githubusercontent.com/1504716/92136214-1fcad600-ede2-11ea-81bb-0ba65c4d93cb.png)

daniloprates reviewed (approved):
Great job!

gnpf commented:
@DeeJayElly The first issue is fixed, but the spinner is still a different color con the [delegation](https://drive.google.com/file/d/1SSEwognNYa9A_PfGPXTyyYD17V0B3zag/view?usp=sharing) and [transaction](https://drive.google.com/file/d/16kAxfSHbEnszVnAr99uUxHhgkxeTK2f7/view?usp=sharing) dialog.

DeeJayElly commented:
> @DeeJayElly The first issue is fixed, but the spinner is still a different color con the [delegation](https://drive.google.com/file/d/1SSEwognNYa9A_PfGPXTyyYD17V0B3zag/view?usp=sharing) and [transaction](https://drive.google.com/file/d/16kAxfSHbEnszVnAr99uUxHhgkxeTK2f7/view?usp=sharing) dialog.

Fixed @gnpf

DeeJayElly commented:
> About this one:
> 
> ![image](https://user-images.githubusercontent.com/1504716/92136387-59034600-ede2-11ea-89dc-346da7ff8a4d.png)
> 
> This is how it's looking on Storybook. Waiting for node sync to check in the actual app, but it's worth having a look:
> 
> ![image](https://user-images.githubusercontent.com/1504716/92136214-1fcad600-ede2-11ea-81bb-0ba65c4d93cb.png)

Yes its different i storybook, that case is probably not handled.

DeeJayElly commented:
> @DeeJayElly please focus on the parts @gnpf reported here [#2152 (comment)](https://github.com/input-output-hk/daedalus/pull/2152#issuecomment-685725138)

@nikolaglumac finally covered all the cases.

gnpf commented:
@DeeJayElly The last fixes look great. There is some new issue with the dropdowns on delegation [slider](https://drive.google.com/file/d/17GexuIddBbaEe0rwoD7XruUaMklMvEmV/view?usp=sharing) and [dialog](https://drive.google.com/file/d/1598DjyFJfct08Fh1EVOA7ZiF6Qa_bH0d/view?usp=sharing).

nikolaglumac commented on source/renderer/app/stores/SidebarStore.js:
```diff
@@ -97,7 +97,7 @@ export default class SidebarStore extends Store {
       [categories.PAPER_WALLET_CREATE_CERTIFICATE.name]: false,
       [categories.STAKING_DELEGATION_COUNTDOWN.name]: isShelleyPending,
-      [categories.STAKING.name]: isShelleyActivated,
+      [categories.STAKING.name]: true,
```
@daniloprates why did you change this? 
During initial syncing while we are still in Byron era this variable should be false?

nikolaglumac replied:
@daniloprates I have reverted this change in https://github.com/input-output-hk/daedalus/commit/adc5aa295feec337f8b22de3712c9211332c767c

daniloprates replied:
My bad, was just to enable the staking icon locally.

nikolaglumac commented:
> @DeeJayElly The last fixes look great. There is some new issue with the dropdowns on delegation [slider](https://drive.google.com/file/d/17GexuIddBbaEe0rwoD7XruUaMklMvEmV/view?usp=sharing) and [dialog](https://drive.google.com/file/d/1598DjyFJfct08Fh1EVOA7ZiF6Qa_bH0d/view?usp=sharing), not sure if this is related to this PR.

@gnpf @DeeJayElly both of these issues are fixed with https://github.com/input-output-hk/daedalus/pull/2152/commits/4ecd1a10ed2945c75e0db4a7ba4153f99a3715ea

nikolaglumac reviewed (approved):
Great work @DeeJayElly 🎉

gnpf commented:
@DeeJayElly @nikolaglumac The spinner on "Rewards" is misaligned (this is not happening on develop and last production build)

<img width="903" alt="Screenshot at Sep 04 11-36-55" src="https://user-images.githubusercontent.com/14169859/92252637-e5763d00-eea4-11ea-8f14-fbc61ea1e52e.png">

a-rukin commented:
![](https://user-images.githubusercontent.com/2539345/92260776-1b391700-eee1-11ea-81cb-e795af2c90cc.png)

can we fit it in one line 😓 ?

a-rukin commented:
![](https://user-images.githubusercontent.com/2539345/92261044-7ec34480-eee1-11ea-9056-10677ca82133.png)

Was it always like this? I mean the 30px padding belongs to scrollable area, not button (it aclually could be a a split 20 + 10) -> now it looks like cropped

DeeJayElly commented:
> ![](https://user-images.githubusercontent.com/2539345/92261044-7ec34480-eee1-11ea-9056-10677ca82133.png)
> 
> Was it always like this? I mean the 30px padding belongs to scrollable area, not button (it aclually could be a a split 20 + 10) -> now it looks like cropped

Fixed. I split them in 2 parts 20px+10px;

DeeJayElly commented:
> ![](https://user-images.githubusercontent.com/2539345/92260776-1b391700-eee1-11ea-81cb-e795af2c90cc.png)
> 
> can we fit it in one line 😓 ?

Fixed

DeeJayElly commented:
> @DeeJayElly @nikolaglumac The spinner on "Rewards" is misaligned (this is not happening on develop and last production build)
> 
> <img alt="Screenshot at Sep 04 11-36-55" width="903" src="https://user-images.githubusercontent.com/14169859/92252637-e5763d00-eea4-11ea-8f14-fbc61ea1e52e.png">

Fixed

DeeJayElly commented:
@nikolaglumac @a-rukin @gnpf All the requested changes have been implemented, this PR is ready to be merged.

a-rukin commented:
Fixes look good to me)
